playground/lib/__tests__/lib.spec.ts

```diff
@@ -33,7 +33,7 @@ describe.runIf(isBuild)('build', () => {
       'dist/nominify/my-lib-custom-filename.iife.js',
     )
     // esbuild helpers are injected inside of the IIFE wrapper
-    expect(code).toMatch(/^var MyLib=function\(\)\{"use strict";/)
+    expect(code).toMatch(/^var MyLib=function\(\)\{(.*?)"use strict";/)
```

Comment:
No statement should be placed before `"use strict"` otherwise, it won't work.

> put the exact statement "use strict"; (or 'use strict';) in the function's body before any other statements.
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#strict_mode_for_functions

It seems the UMD one has the same problem. I think we need to fix this.

Reply:
This seems to be a bug in esbuild.
https://esbuild.github.io/try/#dAAwLjE5LjIALS10YXJnZXQ9ZXMyMDE1AGNvbnN0IGZvbyA9ICgpID0+IHsKICd1c2Ugc3RyaWN0JzsKICAoZnVuY3Rpb24gKCkge30pKCk/LmZvbzsKICAoKCkgPT4ge30pKCk/LmZvbzsKCiAgY29uc29sZS5sb2cod2luZG93LmJhciA/PyAnZm9vJyk7Cn07Cgpjb25zdCBiYXIgPSBmdW5jdGlvbiAoKSB7CiAgJ3VzZSBzdHJpY3QnOwogIChmdW5jdGlvbiAoKSB7fSkoKT8uZm9vOwogICgoKSA9PiB7fSkoKT8uZm9vOwoKICBjb25zb2xlLmxvZyh3aW5kb3cuYmFyID8/ICdmb28nKTsKfTsKCg

I'm not sure what's the best way to handle this on our side.

Reply:
@sapphi-red Thanks for the review! Since this esbuild issue has been around unnoticed for quite a long time, it probably didn't cause any issues at runtime. Maybe we can just add a comment to this test case referencing the esbuild issue for now, and update it later in the future once esbulid fixes it?

Reply:
That makes sense 👍